example/echo-op/echo_op.cpp

```diff
@@ -13,13 +13,24 @@
 
 //[example_core_echo_op_1
 
+template<
+    class AsyncStream,
+    class CompletionToken>
+auto
+async_echo(AsyncStream& stream, CompletionToken&& token)
+
+//]
+    -> beast::async_return_type<CompletionToken, void(beast::error_code)>;
```

Comment:
Right, this ensures that all snippets compile. I think `auto` looks better early in the section.